doc/build-osx.md

```diff
@@ -13,97 +13,108 @@ When the popup appears, click `Install`.
 
 Then install [Homebrew](http://brew.sh).
 
-Dependencies
-----------------------
+## Dependencies
 
-    brew install automake berkeley-db4 libtool boost --c++11 miniupnpc openssl pkg-config homebrew/versions/protobuf260 --c++11 qt5 libevent curl
+    brew install automake berkeley-db4 libtool boost --c++11 miniupnpc openssl pkg-config homebrew/core/protobuf260 --c++11 qt5 libevent curl
 
 NOTE: Building with Qt4 is still supported, however, could result in a broken UI. Building with Qt5 is recommended.
 
-Build NavCoin Core
-------------------------
+## Build NavCoin Core
```

Comment:
@matt-auckland should we insert a step here for running `make` from `/depends` ? Also maybe add some notes about what needs to be on user's PATH for proper compilation, QT flags...

Reply:
See step 2 of this section 😛 Lines 31-33

Reply:
That said the fact you didn't see it where you expected means it's not well written